python_multipart/multipart.py

```diff
@@ -221,11 +223,13 @@ class Field:
 
     Args:
         name: The name of the form field.
+        content_type: The value of the Content-Type header for this field.
     """
 
-    def __init__(self, name: bytes | None) -> None:
+    def __init__(self, name: bytes, content_type: str | None = None) -> None:
```

Comment:
Why did the `name` type changed here? Now we have a mismatch between this and the `File.field_name`. 🤔 

Can we revert this bit?